spec/money_spec.rb

```diff
@@ -181,29 +181,35 @@
     end
 
     it "accepts an optional currency" do
```

Comment:
You should probably split these tests into context — with a `default_currency` set and with it being nil, because the behaviour is different. From what I can tell right now a case when a `default_currency` is `nil` and you initialize it without a currency is untested.

Reply:
> You should probably split these tests into context — with a `default_currency` set and with it being nil, because the behaviour is different. From what I can tell right now a case when a `default_currency` is `nil` and you initialize it without a currency is untested.

_these tests_ being, for `Money.from_amount` or for others such as `Money#amount` etc? 🤓
I have added two more contexts in `.from_amount`'s block.